doc/guides/linux_gsg/linux_gsg.rst

```diff
@@ -145,7 +145,16 @@ On a NUMA machine with two nodes, pages should be allocated explicitly on separa
    For 1GB pages, it is not possible to reserve the hugepage memory after the system has booted.
 
 Prerequisites
-~~~~~~~~~~~~~~
+~~~~~~~~~~~~~
+
+Building CNDP from behind a proxy server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If building behind a proxy server the following is required before excuting a git clone or other commands needs to pass thru the proxy server.
+
+.. code-block:: console
+
+   https_proxy="password@proxy.server:port>https://user:password@proxy.server:port" git clone https://github.com/libbpf/libbpf.git
```

Comment:
This isn't right. Should be
```
export http_proxy="http://proxy.server:port"
export https_proxy="https://proxy.server:port"
```

And you can drop the "git clone" part which is irrelevant.

Reply:
We need to drop the proxy setup as the developer should already know how to setup his proxy and we should not be trying to teach them how to setup a proxy.

Just extra text and most likely will not apply to every possible proxy setup.